openapi/anchor-platform/bundled-callbacks.yaml

```diff
@@ -1064,8 +1090,8 @@ components:
           $ref: '#/components/schemas/Amount'
         amount_out:
           $ref: '#/components/schemas/Amount'
-        amount_fee:
```

Comment:
Should we remove this field? I know it's deprecated in 2.9 and we removed deprecated fields from the docs in the past but now that we have versioned docs, I think we should keep it. WDYT?

Reply:
Yes we always remove deprecated fields from docs. This `amount_fee` is totally removed from platform transaction in 3.0 anyways.

Reply:
I think we should not remove deprecated fields now that we have versioning but I'm okay with merging as is.

Reply:
" can wait till after the 2.9 release to merge it. In this way it will goes to 3.0

Reply:
We decided to make `next`, 2.9 for now. I don't think we will have a 2.10 so next will point to 3.0 after this is merged.